src/structures/Presence.js

```diff
@@ -17,8 +17,18 @@ const { ActivityTypes } = require('../util/Constants');
 class Presence {
   constructor(client, data = {}) {
     Object.defineProperty(this, 'client', { value: client });
+    /**
+     * The userID of this presence
+     * @type {string}
+     */
     this.userID = data.user.id;
+
+    /**
+     * The guild of this presence
+     * @type {Guild}
```

Comment:
Can be undefined (Client#user)
```suggestion
     * @type {?Guild}
```